docs/recipes/build/remove-fake.md

```diff
@@ -0,0 +1,52 @@
+# How do I remove the use of FAKE?
+[FAKE](https://fake.build/) is a tool for build automation. The standard SAFE template comes with a [ready-made build project](/template-safe-commands) at the root of the solution that provides support for many common SAFE tasks.
+
+If you would prefer not to use FAKE, you can of course simply ignore it, but this recipes shows how to completely remove it from your repository. It is important to note that having removed FAKE, you will have to follow a more manual approach to each of these processes. This recipe will only include instructions on how to build and deploy the application after removing FAKE.
+
+> Note that the minimal template does not use FAKE by default, and **this recipe only applies to the standard template**.
+
+#### 1. Build project
+Delete `Build.fs`, `Build.fsproj`, `Helpers.fs` at the root of the solution.
+#### 2. Dependencies
+Find the following dependencies inside the `paket.dependencies` file that’s also at the root of the solution and delete them.
+```fsharp
+nuget Fake.Core.Target
+nuget Fake.IO.FileSystem
+nuget Farmer
+```
+
+Also remove the `paket.references` at the root of the solution which should contain:
+```
+Fake.Core.Target
+Fake.IO.FileSystem
+Farmer
+```
+
+#### 3. Paket Install
+Then, execute `paket install` in your terminal at the root of the solution. This will remove the dependencies that the Build project was using.
+
+## Running the App
```

Comment:
Is this section not shown for how to run the app generated by the minimal template option? If so, don't duplicate here, just link to that.

Reply:
I don't think there is a section showing how to run the minimal app but I do think it's in a few places. Would it be better to put this in a new section and reference that in all the places we show how to run the minimal app?

Reply:
Yes, definitely.

Reply:
But that should be done as a separate PR I think.